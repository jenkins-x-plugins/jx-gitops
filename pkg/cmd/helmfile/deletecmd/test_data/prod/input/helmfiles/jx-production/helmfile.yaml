filepath: ""
environments:
  default:
    values:
    - jx-values.yaml
namespace: jx-production
repositories:
- name: dev
  url: http://chartmuseum-jx.35.242.181.72.nip.io/
releases:
- chart: dev/another
  version: 0.0.1
  name: another
  values:
  - jx-values.yaml
- chart: dev/cheese
  version: 0.0.73
  name: cheese
  values:
  - jx-values.yaml
templates: {}
renderedvalues: {}
