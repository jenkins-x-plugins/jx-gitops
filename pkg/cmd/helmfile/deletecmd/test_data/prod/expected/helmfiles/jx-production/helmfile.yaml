filepath: ""
environments:
  default:
    values:
    - jx-values.yaml
namespace: jx-production
repositories:
- name: dev
  url: http://chartmuseum-jx.35.242.181.72.nip.io/
releases:
- chart: dev/another
  version: 0.0.1
  name: another
  values:
  - jx-values.yaml
templates: {}
renderedvalues: {}
